source/classes.tex

```diff
@@ -2496,7 +2496,7 @@
 its \grammarterm{declarator} shall be
 a function declarator\iref{dcl.fct} of the form
 \begin{ncsimplebnf}
-ptr-declarator \terminal{(} parameter-declaration-clause \terminal{)} \opt{cv-qualifier-seq}
+ptr-declarator \terminal{(} parameter-declaration-clause \terminal{)} \opt{cv-qualifier-seq}\br
```

Comment:
Is this change in the wrong commit?

Reply:
Yes, this should be squashed into "CWG1726 Declarator operators and conversion functions".

If you want to, I can fix that and force-push.

Reply:
fixed